### PR TITLE
add IR module activation in rc.local

### LINKED ISSUE #133: [Ansible] IR missing module activation in /etc/rc.local
```
    rendre permanent en  ajouter une ligne dans /etc/rc.local

.....
.....
  printf "My IP address is %s\n" "$_IP"
fi
ir-keytable -a /etc/rc_maps.cfg -s rc0
exit 0
```

### PR DESCRIPTION
**Pull request by yannis-mlgrn**
Fixes #133 

## ✨ Features :
- ✨ add IR module activation of rc.local in the ansible recipe

### CODE REVIEW

**Comment by yannis-mlgrn** [comment]
I have never used this ansible module and I have don't test it , If test pass @Benvii you should call me the king of ansible

**Comment by yannis-mlgrn** [comment]
why git has break my test ?? 
``` 
    arm-image.raspbian: fatal: [127.0.0.1]: FAILED! => {"cache_update_time": 1655588521, "cache_updated": false, "changed": false, "msg": "'/usr/bin/apt-get -y -o \"Dpkg::Options::=--force-confdef\" -o \"Dpkg::Options::=--force-confold\"       install 'git'' failed: E: Failed to fetch http://raspbian.mirrors.lucidnetworks.net/raspbian/pool/main/g/git/git-man_2.20.1-2+deb10u3_all.deb  Temporary failure resolving 'raspbian.mirrors.lucidnetworks.net'\nE: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?\n", "rc": 100, "stderr": "E: Failed to fetch http://raspbian.mirrors.lucidnetworks.net/raspbian/pool/main/g/git/git-man_2.20.1-2+deb10u3_all.deb  Temporary failure resolving 'raspbian.mirrors.lucidnetworks.net'\nE: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?\n", "stderr_lines": ["E: Failed to fetch http://raspbian.mirrors.lucidnetworks.net/raspbian/pool/main/g/git/git-man_2.20.1-2+deb10u3_all.deb  Temporary failure resolving 'raspbian.mirrors.lucidnetworks.net'", "E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?"], "stdout": "Reading package lists...\nBuilding dependency tree...\nReading state information...\nThe following additional packages will be installed:\n  git-man libcurl3-gnutls liberror-perl\nSuggested packages:\n  git-daemon-run | git-daemon-sysvinit git-doc git-el git-email git-gui gitk\n  gitweb git-cvs git-mediawiki git-svn\nThe following NEW packages will be installed:\n  git git-man libcurl3-gnutls liberror-perl\n0 upgraded, 4 newly installed, 0 to remove and 0 not upgraded.\nNeed to get 6137 kB of archives.\nAfter this operation, 32.9 MB of additional disk space will be used.\nGet:1 http://mirror.pit.teraswitch.com/raspbian/raspbian buster/main armhf libcurl3-gnutls armhf 7.64.0-4+deb10u2 [292 kB]\nGet:2 http://mirrors.gigenet.com/raspbian/raspbian buster/main armhf liberror-perl all 0.17027-2 [30.9 kB]\nErr:3 http://raspbian.mirrors.lucidnetworks.net/raspbian buster/main armhf git-man all 1:2.20.1-2+deb10u3\n  Temporary failure resolving 'raspbian.mirrors.lucidnetworks.net'\nGet:4 http://raspbian.mirrors.lucidnetworks.net/raspbian buster/main armhf git armhf 1:2.20.1-2+deb10u3 [4194 kB]\nFetched 4517 kB in 15s (306 kB/s)\n", "stdout_lines": ["Reading package lists...", "Building dependency tree...", "Reading state information...", "The following additional packages will be installed:", "  git-man libcurl3-gnutls liberror-perl", "Suggested packages:", "  git-daemon-run | git-daemon-sysvinit git-doc git-el git-email git-gui gitk", "  gitweb git-cvs git-mediawiki git-svn", "The following NEW packages will be installed:", "  git git-man libcurl3-gnutls liberror-perl", "0 upgraded, 4 newly installed, 0 to remove and 0 not upgraded.", "Need to get 6137 kB of archives.", "After this operation, 32.9 MB of additional disk space will be used.", "Get:1 http://mirror.pit.teraswitch.com/raspbian/raspbian buster/main armhf libcurl3-gnutls armhf 7.64.0-4+deb10u2 [292 kB]", "Get:2 http://mirrors.gigenet.com/raspbian/raspbian buster/main armhf liberror-perl all 0.17027-2 [30.9 kB]", "Err:3 http://raspbian.mirrors.lucidnetworks.net/raspbian buster/main armhf git-man all 1:2.20.1-2+deb10u3", "  Temporary failure resolving 'raspbian.mirrors.lucidnetworks.net'", "Get:4 http://raspbian.mirrors.lucidnetworks.net/raspbian buster/main armhf git armhf 1:2.20.1-2+deb10u3 [4194 kB]", "Fetched 4517 kB in 15s (306 kB/s)"]}
```

**Comment by viandoxdev** [comment]
Yeah that happens, just restart the action

**Review by Benvii — CHANGES_REQUESTED** [review]
Final rc.local file is : 
```
#!/bin/sh -e
#
# rc.local
#
# This script is executed at the end of each multiuser runlevel.
# Make sure that the script will "exit 0" on success or any other
# value on error.
#
# In order to enable or disable this script just change the execution
# bits.
#
# By default this script does nothing.

# Print the IP address
_IP=$(hostname -I) || true
if [ "$_IP" ]; then
  printf "My IP address is %s\n" "$_IP"
fi

exit 0
# BEGIN ANSIBLE MANAGED BLOCK
printf "My IP address is %s\n" "$_IP"
fi
ir-keytable -a /etc/rc_maps.cfg -s rc0
exit 0
# END ANSIBLE MANAGED BLOCK

```

Will not work

**Comment by yannis-mlgrn** [comment]
It's the same text like in the issue
```
  printf "My IP address is %s\n" "$_IP"
fi
ir-keytable -a /etc/rc_maps.cfg -s rc0
exit 0
```
The indentation before printf is important ???

**Comment by Benvii** [comment]
The script will end at first exit 0 (that what exist is made for), so ir-keytable command will never be executed, your lines are dead code (they will never run)

**Comment by yannis-mlgrn** [comment]
mhhh ok

**Comment by Benvii** [comment]
As you have more time than me, can you check the result file by mounting the build image (see how to do it in packer/readme.md), @viandoxdev as documented it

**Comment by Benvii** [comment]
/etc/rc.local looks like this now :
```
#!/bin/sh -e
#
# rc.local
#
# This script is executed at the end of each multiuser runlevel.
# Make sure that the script will "exit 0" on success or any other
# value on error.
#
# In order to enable or disable this script just change the execution
# bits.
#
# By default this script does nothing.

# Print the IP address
_IP=$(hostname -I) || true
if [ "$_IP" ]; then
  printf "My IP address is %s\n" "$_IP"
fi

# BEGIN ANSIBLE MANAGED BLOCK
printf "My IP address is %s\n" "$_IP"
fi
ir-keytable -a /etc/rc_maps.cfg -s rc0
# END ANSIBLE MANAGED BLOCK
exit 0

```

There is a fi in the middle of no-where.

File: ansible/roles/IR-lap-timer/tasks/main.yml [hunk 1]
```diff
@@ -6,9 +6,19 @@
...
     group: root
 
+- name: Add IR module activation in rc.local
+  become: yes
+  blockinfile:
+    dest: /etc/rc.local
+    insertbefore: '^exit 0'
+    block: |
+      printf "My IP address is %s\n" "$_IP"
+      fi
```
Comment (Benvii) [inline ansible/roles/IR-lap-timer/tasks/main.yml:16]: Remove line 15 and 16, we only need ir-keytable command

**Review by Benvii — CHANGES_REQUESTED** [review]

**Comment by yannis-mlgrn** [comment]
fixed @Benvii

**Comment by Benvii** [comment]
Putting this command in rc.local works : `ir-keytable -v -p nec` it could be a workaround. Can you change it @yannis-mlgrn 

@landerneRPi can you help here ?

**Review by Benvii — APPROVED** [review]
Test and working.

**Pull request merged**